src/main/java/gov/nasa/podaac/swodlr/security/AbstractJweCookieStore.java

```diff
@@ -211,4 +222,11 @@ protected static SwodlrSecurityProperties getSecurityProperties() {
     }
     return securityProperties;
   }
+
+  private static SwodlrProperties getSwodlrProperties() {
```

Comment:
Does this need to be a static lookup instead of injected? It's hard to tell from the code snippets but it looks like it's only called from a non-static context.

Reply:
It does - `AbstractJweCookieStore` is extended by `JweCookieAuthorizedClientStore` and `JweSession`; both of which are managed as part of the request lifecycle instead of as Spring beans. They only last the duration of a request.

As for why it's static - that's just a small optimization I made since the properties objects weren't changing between each instance.